osu.Game.Tournament/TournamentGameBase.cs

```diff
@@ -241,9 +245,7 @@ private bool addRoundBeatmaps()
             {
                 var b = beatmapsRequiringPopulation[i];
 
-                var req = new GetBeatmapRequest(new APIBeatmap { OnlineID = b.ID });
-                API.Perform(req);
-                b.Beatmap = new TournamentBeatmap(req.Response ?? new APIBeatmap());
+                b.Beatmap = new TournamentBeatmap(beatmapCache.GetBeatmapAsync(b.ID).GetAwaiter().GetResult() ?? new APIBeatmap());
```

Comment:
Is this `GetAwaiter().GetResult()` stuff an attempt to bypass the fact that `.Result` is banned? If yes, then did you try calling `.GetResultSafely()` which is a framework extension for this express purpose?

Reply:
yes, this should be my oversight, sorry

already fix